File: FolsomiaSystem.Api/App/FolsomiaSystem.Domain/Entities/AuditLog.cs
using FolsomiaSystem.Domain.Enums;
using System;

namespace FolsomiaSystem.Domain.Entities
{
    public class AuditLog
    {
        public int AuditLogId { get; set; }
        public string MessageLog { get; set; }
        public DateTime DateLog { get; set; }
        public OperationLog OperationLog { get; set; }
        public StatusLog StatusLog { get; set; }

    }
}

File: FolsomiaSystem.Angular/App/FolsomiaSystem-App/src/app/theme/shared/models/audit-log.ts
import { OperationLog } from '../enum/operation-log';
import { StatusLog } from '../enum/status-log';

export class AuditLog{
    auditLogId:number;
    messageLog:string;
    dateTime:string;
    operationLog:OperationLog;
    statusLog:StatusLog;




}